webpack/assets/javascripts/react_app/components/ModelsTable/ModelsTable.test.js

```diff
@@ -1,23 +1,29 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { shallow, mount } from 'enzyme';
```

Comment:
Do we still need the `shallow` here?

Reply:
yes, it is still used in the second test